### PR TITLE
[WIP] replaced espresso spce tcl script by python version.

### LINKED ISSUE #11: Espresso tutorial needs an update from tcl to python
Bascially the `spce.tcl` in https://github.com/votca/csg-tutorials/tree/master/spce/ibi_espresso needs to be converted to python.

### PR DESCRIPTION
**Pull request by fweik**
This is not complete yet: The convention for the force tables changed, in the tcl version there was an implicit factor `1/r` (c.f. 5.1.1. of [the User's Guide](http://espressomd.org/wordpress/wp-content/uploads/2017/06/ESPResSo_3.3.1_UsersGuide.pdf)), which is no longer there in the python version. So probably the output of votca has to be adapted. It would also probably make sense to pack the tables together with the range into a xml or something so that it can be read automatically.

Also I changed the output format to h5md, which Espresso has native support for and which is also supported by votca. I now use MDAnalysis to read in the topology, if that external dependency should be avoided Espresso can read pdb files (but funny enough it can not read h5md files, which it can write).
The self-coded gro parser seemed brittle to me, but of course you can also put that back in.

Eventually fixes #11.

### CODE REVIEW

**Comment by junghans** [comment]
Once https://bodhi.fedoraproject.org/updates/FEDORA-2018-3a0a92e596 gets in, I will make the espresso test part of the IC.

**Comment by junghans** [comment]
Note to myself add some hackery here: https://github.com/votca/csg/blob/master/share/scripts/inverse/run_genericsim.sh#L44

File: spce/ibi_espresso/spce.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+print("Program Information:\n{}\n".format(espressomd.features()))
+
+# set system properties:
+skin = 0.5
+time_step = 0.002
+friction = 5.0
+int_steps = 900
+eq_steps = 100
+
+in_atoms = mda.Universe('spce.gro')
```
Comment (junghans) [inline spce/ibi_espresso/spce.py]: @fweik do you support a file format. e.g. pdb, we could read without MDanalysis? MDanalysis just seems a bit heavy.

**Comment by fweik** [comment]
Sorry, I forgot about this. There seems to be a bug. Will investigate.

**Comment by junghans** [comment]
Ping @fweik

**Comment by junghans** [comment]
Huh, is this fixed?

**Comment by fweik** [comment]
No, but it diverged too much. I found the issue in Espresso and make a new
PR once it is fixed.

**Comment by junghans** [comment]
Ok, you can cherry-pick e9727d01acab6fc94e9014a52dc798f0247bb0be over!

**Pull request closed without merging**